streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/TopologyTest.scala

```diff
@@ -146,18 +142,17 @@ class TopologyTest extends JUnitSuite {
 
       val userRegionsTable: KTable[String, String] = builder.table(userRegionsTopic)
 
-      val clicksPerRegion: KTable[String, Long] =
-        userClicksStream
-          .leftJoin(userRegionsTable)((clicks, region) => (if (region == null) "UNKNOWN" else region, clicks))
-          .map((_, regionWithClicks) => regionWithClicks)
-          .groupByKey
-          .reduce(_ + _)
+      userClicksStream
```

Comment:
This variable was unused.

Reply:
I think this was intentional to describe the operation. The warning is really annoying though and I've debated removing that variable several times. :)

Reply:
Yeah, the thought crossed my mind. I just didn't think the description was that necessary. Although, I guess I can add a comment instead...